hoomd/md/ConstantForceCompute.cc

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2009-2022 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#include "ConstantForceCompute.h"
+#include "hoomd/RNGIdentifiers.h"
+#include "hoomd/RandomNumbers.h"
+
+#include <vector>
+
+namespace hoomd
+    {
+namespace md
+    {
+/*! \file ConstantForceCompute.cc
+    \brief Contains code for the ConstantForceCompute class
+*/
+
+/*! \param rotation_diff rotational diffusion constant for all particles.
+ */
+ConstantForceCompute::ConstantForceCompute(std::shared_ptr<SystemDefinition> sysdef,
+                                           std::shared_ptr<ParticleGroup> group)
+
+    : ForceCompute(sysdef), m_group(group), m_need_rearrange_forces(false)
+    {
+    // allocate memory for the per-type constant_force storage and initialize them to (1.0,0,0)
+    GlobalVector<Scalar3> tmp_f_constantVec(m_pdata->getNTypes(), m_exec_conf);
```

Comment:
```suggestion
    GlobalVector<Scalar3> tmp_f(m_pdata->getNTypes(), m_exec_conf);
```
This variable name mixes naming conventions. I think `tmp_f` is sufficiently descriptive.